compiler/one-cmds/how-to-create-hdf5-dataset.txt

```diff
@@ -0,0 +1,96 @@
+About
+-----
+
+Last update: 2020-11-06
+
+This document briefly explains how to create an input dataset for one-quantize.
+
+The input dataset of one-quantize has the form of hdf5.
+For users who are not familiar with hdf5, we provide a tool to convert raw data to hdf5.
+
+Workflow to generate input dataset (hdf5 file)
+1. Pre-process input data for the target model and save them as raw data files
+2. Package the raw data files into the hdf5 file using rawdata2hdf5
+
+Note: Users should prepare raw data which can be fed to the target model.
+This is because we don't know which pre-processing logic was used for the target model.
+
+rawdata2hdf5
+---------------
+
+rawdata2hdf5 is a tool to package raw data files into an hdf5 file,
+which is the format of input dataset for one-quantize.
+
+Usage:  rawdata2hdf5 --data_list <path/to/text/file> --output_path <path/to/output/file>
+
+Example
+---------------
+
+Let's make an input dataset for InceptionV3 model.
+
+1. Download sample images (You can use your own dataset)
+
+$ wget https://github.com/Samsung/ONE/files/5499172/img_files.zip
+$ unzip img_files.zip
+$ tree img_files
+img_files
+├── bald-eagle.jpg
+├── cow.jpg
+├── deer-in-wild.jpg
+├── fox.jpg
+├── ladybird.jpg
+├── orange-portocaliu.jpg
+├── pink-lotus.jpg
+├── red-church.jpg
+├── tomatoes.jpg
+└── young-dachshund.jpg
```

Comment:
Is there no problem with copyrights with the image?

Reply:
Yes. I downloaded them from [freeimages.com](https://freeimages.com). The photos are free if we do not use them for resale. Here is the full [content lincense agreement](https://www.freeimages.com/photo/eagle-1523807).